### PR TITLE
Issue189 Redirect to SEB config link if using SEB

### PR DESCRIPTION
**Pull request by dmitriim**

### CODE REVIEW

File: rule.php [hunk 1]
```diff
@@ -563,4 +567,22 @@ private function should_display_download_seb_link() : bool {
...
+     */
+    private function redirect_to_seb_config_link() {
+        global $PAGE;
+
+        $seblink = \quizaccess_seb\link_generator::get_link($this->quiz->cmid, true, is_https());
+        $PAGE->requires->js_amd_inline("document.location.replace('" . $seblink . "')");
+    }
+
+    /**
+     * Check if we need to redirect to SEb config link.
```
Comment (nhoobin) [inline rule.php]:
```suggestion
     * Check if we need to redirect to SEB config link.
```

File: tests/phpunit/rule_test.php [hunk 2]
```diff
@@ -1193,4 +1195,28 @@ public function test_blocks_display_after_attempt_finished() {
...
+    public function test_should_redirect_to_seb_config_link() {
+        $this->setAdminUser();
+        $this->quiz = $this->create_test_quiz($this->course, settings_provider::USE_SEB_CONFIG_MANUALLY);
+
+        $reflection = new \ReflectionClass('quizaccess_seb');
+        $method = $reflection->getMethod('should_redirect_to_seb_config_link');
+        $method->setAccessible(true);
+
+        set_config('autoreconfigureseb', '0', 'quizaccess_seb');
+        $_SERVER['HTTP_USER_AGENT'] = 'NOT SEB';
```
Comment (nhoobin) [inline tests/phpunit/rule_test.php]: Is this correct here? As is_using_seb calls strpos($_SERVER['HTTP_USER_AGENT'], 'SEB')

Comment (dmitriim) [reply tests/phpunit/rule_test.php]: good catch

Comment (dmitriim) [reply tests/phpunit/rule_test.php]: fixed

**Review by nhoobin — APPROVED** [review]

**Pull request merged**